deployment/network-operator/README.md

```diff
@@ -216,6 +216,20 @@ Specifies components to deploy in order to facilitate a secondary network in Kub
 | `ipamPlugin.repository` | string | `dougbtv` | IPAM CNI Plugin image repository  |
 | `ipamPlugin.version` | string | `v0.3` | IPAM CNI Plugin image version  |
 
+## Helm Tests
+
+Network Operator has helm tests to verify deployment, currently supported tests:
+- Device Plugin Resource: This test create a pod that requests the first resource in `devicePlugin.resources`
+- RDMA Traffic: This test creates a pod that run server and client with `rping`
+
+Run the helm test with following command after deploying network operator with helm
+```
+$ helm test -n network-operator network-operator
+```
+Notes:
+- Test will keeping running endlessly if pod creating failed so it is recommended to use `--timeout` which fails test after exceeding given timeout
+- Default PF to run test is `ens2f0` to override it add `--set test.pf=<pf_name>` to `helm install/upgrade`
```

Comment:
can we avoid the default and make it as mandatory args?

Reply:
yes, when deployed without setting error will appear
```
Error: template: network-operator/templates/tests/test-rping.yaml:9:21: executing "network-operator/templates/tests/test-rping.yaml" at <.Values.test.pf>: nil pointer evaluating interface {}.pf
```

Reply:
if we go with the mandatory approach, this comment should state that it is mandatory AND  example command above should reflect that as well.

Reply:
It was decided not to go with the mandatory approach